### ISSUE TITLE
Full width should support an empty string

### ISSUE THREAD

**Issue by Alexnortung** [mjmlio/mjml#2008]
# Problem

If I decide that all my `mj-section`s should be full-width, but later I decide that one of my sections should not, then I would have to add full-width to all my sections (or give them a class, same same).

It already seems to work fine in the finished HTML, but it's just annoying that I get the error

# Solution

I would like to add `full-width=""` or `full-width="false"` to my `mj-section`, without the compiler giving me a warning.

# Additional info

The error I get

```
Line 58 of path/index.mjml (mj-section) — Attribute full-width has invalid value:  for type Enum, only accepts full-width
```

**Comment by iRyusa** [comment]
Why don't you remove the attribute then ? I don't really understand the need here to add new value(s) ? 

We would love to have something similar as empty attribute in HTML spec but we didn't test if htmlparser2 supports them https://html.spec.whatwg.org/multipage/syntax.html#syntax-attr-empty

**Comment by Alexnortung** [comment]
I mean the problem is that I defined the full-width attribute in the head inside mj-attributes
Something like this:

```
<mj-head>
<mj-attributes>
<mj-section full-width="full-width" />
...
```
Then I started coding adding dozens of sections which are now all full-width.

However at some point I wish that one of them should not be full-width. So this is not really doable unless you remove it from the head and then apply it to ALL the other sections and the next ones to come.

Hope it makes sense
